cost-analyzer/values.yaml

```diff
@@ -2235,6 +2235,52 @@ kubecostDeployment:
   labels: {}
   annotations: {}
 
+  ## QueryServiceReplicas
```

Comment:
Again, not sure if you meant to add back in QueryServiceReplicas here?

Reply:
Removed, nice catch.